lib/aasm/core/invokers/literal_invoker.rb

```diff
@@ -30,12 +30,21 @@ def exec_subject
           raise(*record_error) unless record.respond_to?(subject, true)
           return record.__send__(subject) if subject_arity.zero?
           return record.__send__(subject, *args) if subject_arity < 0
```

Comment:
The problem seems to come from line 32; `subject_arity` is -1 for a subject like below:
```
def set_follow_up_date(date: nil)
end
```